storybook/src/elements/nuxeo-video/nuxeo-video-info.stories.js

```diff
@@ -0,0 +1,26 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-video/nuxeo-video-info';
+import DocumentBuilder from '../../data/documents.data';
+import videoProperties from '../../data/video.data.js';
+
+storiesOf('UI/nuxeo-video', module).add('Nuxeo Video Info', () => {
+  const documentBuilder = new DocumentBuilder()
+    .setType('File')
+    .setProperties({ 'vid:info': videoProperties['vid:info'] });
+  documentBuilder.build();
+  return html`
+    <style>
+      *[role='widget'] {
+        display: block;
+        padding: 16px;
+        margin: 20px;
+        box-shadow: 0 3px 5px rgba(0, 0, 0, 0.04) !important;
+        font-family: var(--nuxeo-app-font);
+        border-radius: 0;
+        background-color: var(--nuxeo-box) !important;
+      }
+    </style>
+    <nuxeo-video-info role="widget" .document="${documentBuilder}"></nuxeo-video-info>
```

Comment:
Please remove the `role="widget"` and we can define the style directly to `nuxeo-video-info`.
Also, since both stories seem to share the same "wrapping style" (i.e. card style), we may want to have both consistent, no? Both wrapped with `nuxeo-card` or both having the style defined. 😉